app/controllers/courses_controller.rb

```diff
@@ -7,6 +7,7 @@ def show
     @course = Course
       .includes(:subjects)
       .includes(site_statuses: [:site])
+      .includes(:sites)
```

Comment:
It's not a huge deal but i think we should be accessing sites through site_statuses. Otherwise, we're getting sites data back twice